api/build.gradle

```diff
@@ -181,10 +181,6 @@ swaggerSources {
             '--model-name-prefix', 'Leonardo'
     ])
 
-    sam swagger3JavaClient("sam.yaml", "sam", [
```

Comment:
I am not super familiar with Swagger. It seems like this Swagger was one that we built around our understanding of SAm, but you are saying that they have a client that we can use instead. Is a client a one for one replacement of swagger, or do you they serve different purposes?

Reply:
(as we discussed)

Both clients are created by running Swagger-codegen on a version of Sam's API YAML definition, found here: https://github.com/broadinstitute/sam/blob/develop/src/main/resources/swagger/api-docs.yaml

Old process: we copy pieces of the Sam YAML file into our repo as needed, and AoU's build scripts run a codegen step to create an AOU->Sam Java client for our app to use

New process: Sam's build scripts run a codegen step to create a Java client for anyone to use, and they publish it to JFrog.  AoU uses this JFrog library to call Sam, so we no longer need to build our own client